helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -1731,12 +1731,14 @@
     "certmanager": {
       "description": "cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates.",
       "type": "object",
-      "additionalProperties": true
+      "additionalProperties": true,
+      "deprecated": true
```

Comment:
The schema docs - `The deprecated keyword is a boolean that indicates that the [instance](https://json-schema.org/learn/glossary#instance) value the keyword applies to should not be used and may be removed in the future.` Since we are removing cert-manager completely here, should we just remove the instance definitions from schema? Deprecated makes it seem like setting the value will still get certmanager installed

Reply:
- I would not recommend hard removing (deleting the json entry) any fields in values.schema.json at this time. The values.schema.json is used in various ways by Helm and providers of Helm products where I would only recommend applying the `deprecated: true` to values we no longer use.

Reply:
Ok, if you are updating the PR to keep the cert-manager dependency, then adding the deprecated annotation sounds good. My request for removing it was contingent upon the removal of cert-manager, which, as I mentioned in the other comment, I couldn't figure out was part of this PR or not.